content/docs/networking/load-balancing.md

```diff
@@ -1,47 +1,99 @@
 ---
-title: "IPv4 Service IP implementation details"
-date: 2022-08-09T15:56:27+02:00
+title: "Load Balancing in Oakestra"
+date: 2023-03-01T12:56:27+02:00
 draft: false
+hidden: false
 ---
 
-In this wiki page, we'll go through the details of the IPv4 implementation of the Service IPs in Oakestra. 
+# Load Balancing in Oakestra
 
-The building blocks are:
+The load balancing in Oakestra consists of a multi-step address conversion protocol using semantic addresses as part of a overlay network.
+On this page we want to achieve:
 
-* Network Overlays
-* Proxy Translation
-* Interest registration
+- Explain the overlay network
+- Explain the semantic addresses
+- Explain proxy conversion protocol
 
-	
-## Network Overlays
-![Overlay Example](/network/Overlay-layers.png)
+## Overlay network
+
+We start by explaining the different network abstraction layers in place and then continue with the semantic addressing.
+
+### Layers
+
+[Overlay Example](/network/Overlay-layers.png)
```

Comment:
Couldnt find the path to PNG. Are you sure this is correct?

Reply:
It's located in /static/network/Overlay-layers.png so it should be there. Otherwise the others should not work either.
Added the /static.